t/security.t

```diff
@@ -0,0 +1,217 @@
+use Mojo::Base -strict;
+use Test::Mojo;
+use Test::More;
+
+use Mojolicious::Lite;
+post '/global' => sub {
+  my $c = shift->openapi->valid_input or return;
+  $c->reply->openapi(200 => {ok => 1});
+  },
+  'global';
+
+post '/simple' => sub {
+  my $c = shift->openapi->valid_input or return;
+  $c->reply->openapi(200 => {ok => 1});
+  },
+  'simple';
+
+post '/fail_or_pass' => sub {
+  my $c = shift->openapi->valid_input or return;
+  $c->reply->openapi(200 => {ok => 1});
+  },
+  'fail_or_pass';
+
+post '/fail_and_pass' => sub {
+  my $c = shift->openapi->valid_input or return;
+  $c->reply->openapi(200 => {ok => 1});
+  },
+  'fail_and_pass';
+
+post '/cache' => sub {
+  my $c = shift->openapi->valid_input or return;
+  $c->reply->openapi(200 => {ok => 1});
+  },
+  'cache';
+
+our %checks;
+plugin OpenAPI => {
+  url      => 'data://main/sec.json',
+  security => {
+    pass1 => sub {
+      my ($c, $def, $scopes, $cb) = @_;
+      $checks{pass1}++;
+      $cb->(1);
+    },
+    pass2 => sub {
+      my ($c, $def, $scopes, $cb) = @_;
+      $checks{pass2}++;
+      $cb->(1);
+    },
+    fail1 => sub {
+      my ($c, $def, $scopes, $cb) = @_;
+      $checks{fail1}++;
+      $cb->(0);
+    },
+    fail2 => sub {
+      my ($c, $def, $scopes, $cb) = @_;
+      $checks{fail2}++;
+      $cb->(0);
+    },
+  },
+};
+
+my $t = Test::Mojo->new;
+subtest 'global' => sub {
```

Comment:
It's just a matter of taste. I argued with @marcusramberg about perltidy for two years, and now I can't live without it... Maybe the same will happen with subtest, but not tonight 👎